app/default_overrides.go

```diff
@@ -43,7 +43,7 @@ type bankModule struct {
 // DefaultGenesis returns custom x/bank module genesis state.
 func (bankModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	metadata := banktypes.Metadata{
-		Description: "The native staking token of the Celestia network.",
+		Description: "The native token of the Celestia network.",
```

Comment:
On second thought I was wondering if this is breaking. I think this string only gets used at genesis creation time so I don't expect it to be breaking.